### ISSUE TITLE
Software icon problem (Windows maybe other OS)

### ISSUE THREAD

**Issue by Neustradamus** [Xyrio/RSSOwlnix#14]
Go on the .exe, look the icon, there is RSSOwl or Eclipse icon.
There was no problem with 2.2.1.
Size problem, rssowl.ico missing?

**Comment by Xyrio** [comment]
im on windows 7 64bit and i have no icon error for the window. 

try to copy the rssowl.ico over to rssowl 2.5.0 and try with a renamed rssowlnix.ico too. does that help you?

**Comment by Neustradamus** [comment]
![rssowl-icon](https://user-images.githubusercontent.com/104737/37807135-242f4114-2e44-11e8-9b1f-127f6b14e781.png)

**Comment by Xyrio** [comment]
and does it help if you copy the rssowl.ico file over?

**Comment by Neustradamus** [comment]
I can contact you by email?
What is your email address, you can remove after from public...

I think it is at compilation there was not all sizes of the icon...

**Comment by Xyrio** [comment]
yes i understand. the exe has indeed that eclipse icon. ill try to fix it.

**Comment by Neustradamus** [comment]
:)

**Comment by Neustradamus** [comment]
Thanks, it is resolved :)
